File: src/targets/node/axios/fixtures/query.js
import axios from 'axios';

const options = {
  method: 'GET',
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value'
};

axios
  .request(options)
  .then(res => console.log(res.data))
  .catch(err => console.error(err));

File: src/targets/swift/urlsession/fixtures/query.swift
import Foundation
#if canImport(FoundationNetworking)
  import FoundationNetworking
#endif

let url = URL(string: "https://httpbin.org/anything?key=value")!
var components = URLComponents(url: url, resolvingAgainstBaseURL: true)!
let queryItems: [URLQueryItem] = [
  URLQueryItem(name: "foo", value: "bar"),
  URLQueryItem(name: "foo", value: "baz"),
  URLQueryItem(name: "baz", value: "abc"),
  URLQueryItem(name: "key", value: "value"),
]
components.queryItems = components.queryItems.map { $0 + queryItems } ?? queryItems

var request = URLRequest(url: components.url!)
request.httpMethod = "GET"

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))
